app/views/rubygems/show.html.erb

```diff
@@ -41,6 +41,13 @@
 
     <%= render :partial => "rubygems/dependencies", :locals => { :dependencies => @latest_version.dependencies.runtime } %>
     <%= render :partial => "rubygems/dependencies", :locals => { :dependencies => @latest_version.dependencies.development } %>
+    <% if @latest_version.dependencies.present? && @latest_version.indexed %>
+      <div class = "l-half--r">
```

Comment:
I think we need `l-half--l` here.

Reply:
Okay, `--l` seems to be left-aligned, let's remove this div. Nest everything from line 42 to single `<div class = "l-half--l">`, change top div in `_dependencies.html.erb` to `t-list__items` and add `push--s` to `link_to` classes.